StorageTests/TestSQLiteHistory.swift

```diff
@@ -1292,7 +1342,8 @@ class TestSQLiteHistory: XCTestCase {
 
             return history.getTopSitesWithLimit(20) >>== { topSites in
                 XCTAssertEqual(topSites.count, 20)
-                XCTAssertEqual(topSites[0]!.guid, "abc\(0)def")
+                XCTAssertEqual(topSites[0]!.guid, "abc\(5)def")
+                XCTAssertEqual(topSites[1]!.guid, "abc\(0)def")
```

Comment:
After my changes to the query. The zeroth site no longer appears as the first one. I dont think this is technically wrong (it appears second now). Both site5 and site0 should have exactly the same frecency score so technically they should be interchangeable?